src/test/java/org/mockito/internal/creation/bytebuddy/SubclassByteBuddyMockMakerTest.java

```diff
@@ -34,6 +35,7 @@ public void is_type_mockable_excludes_primitive_wrapper_classes() {
     }
 
     @Test
+    @Ignore("Broken on JDK 17")
```

Comment:
We were only running this on 17, but it seems to be failing anyways 😢

Reply:
Fixed the test!